### ISSUE TITLE
Crash on name change attempt

### ISSUE THREAD

**Issue by ShortDevelopment** [nearby-sharing/android#108]
### Android Versions
- Android 10
- Android 11

### Message
```
java.lang.IllegalStateException: Dialog view must contain an EditText with id @android:id/edit
```

**Comment by ShortDevelopment** [comment]
https://stackoverflow.com/a/41510232/15213858

**Comment by ShortDevelopment** [comment]
This bug should now finally be fixed (4bb3d3eef29350734c016d0e0a939f2feb362db7).
An update (`v1.6.1`) is rolling out in preview and will be pushed to production soon.
